src/index.html

```diff
@@ -14,8 +14,17 @@
   <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
   <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.3/js/bootstrap.min.js" integrity="sha384-a5N7Y/aK3qNeh15eJKGWxsqtnX/wWdSZSKp+81YjTmS15nvnvxKHuzaWwXHDli+4" crossorigin="anonymous"></script>
+  <style type="text/css">
+    .fa-spin {
+      margin:300px 700px;
```

Comment:
By setting hard coded margin the spinner may appear at different places in different browsers or resolutions. It may not always appear in the horizontal center of the screen.
![screenshot from 2018-08-29 19-05-28](https://user-images.githubusercontent.com/24729340/44791541-74b62f00-abbf-11e8-9ac5-86880678fd9a.png)

Reply:
It has been changed to this .

![img_20180829_193126](https://user-images.githubusercontent.com/33368759/44792715-26565f80-abc2-11e8-93b0-da29a1386621.jpg)

Reply:
Why 40% from left and top?? @debck

Reply:
@knrt10  made those changes